src/js/routes/jobs.js

```diff
@@ -1,22 +1,23 @@
-import { Redirect, IndexRoute, Route } from "react-router";
 /* eslint-disable no-unused-vars */
 import React from "react";
-/* eslint-enable no-unused-vars */
-
-import JobDetailPage from "../pages/jobs/JobDetailPage";
-import JobsPage from "../pages/JobsPage";
-import JobsTab from "../pages/jobs/JobsTab";
-import JobsTaskDetailPage from "../pages/jobs/JobTaskDetailPage";
+import { IndexRoute, Redirect, Route } from "react-router";
 import TaskDetailsTab
   from "../../../plugins/services/src/js/pages/task-details/TaskDetailsTab";
 import TaskFileBrowser
   from "../../../plugins/services/src/js/pages/task-details/TaskFileBrowser";
-import TaskFilesTab
-  from "../../../plugins/services/src/js/pages/task-details/TaskFilesTab";
 import TaskFileViewer
   from "../../../plugins/services/src/js/pages/task-details/TaskFileViewer";
+import TaskFilesTab
+  from "../../../plugins/services/src/js/pages/task-details/TaskFilesTab";
 import TaskLogsContainer
   from "../../../plugins/services/src/js/pages/task-details/TaskLogsContainer";
+import JobsPage from "../pages/JobsPage";
+/* eslint-enable no-unused-vars */
+import {
+  JobDetailPageContainer as JobDetailPage
+} from "../pages/jobs/JobDetailPageContaiener";
```

Comment:
oops, I think this should be JobDetailPageContainer.js

Reply:
I think that is the correct import, but that the as is a little confusing for me too. I have no strong opinion here so I am approving, but maybe it makes more sense to just not use JobDetailPage and have JobDetailPageContainer everywhere on this file.

Reply:
Ohh, the typo ;)